### ISSUE TITLE
Use .vfsforgit instead of .gvfs directories for GVFS provider

### ISSUE THREAD

**Issue by chrisd8088** [github/VFSForGit#9]
In order to avoid conflicts with GNOME VFS (GVfs) and pre-existing `.gvfs` directories on Linux systems -- not to mention user confusion -- we need the GVFS provider to create and use `.vfsforgit*` hidden directories instead of its current `.gvfs*` ones.  This could be exclusive to the Linux client, though.

See Microsoft/VFSForGit#987.

**Comment by kivikakk** [comment]
https://github.com/microsoft/VFSForGit/pull/1084 will pave the way for this.

**Comment by kivikakk** [comment]
Assigning myself -- I'll take care of the rebase to use the new functionality once microsoft#1084 is merged!

**Comment by kivikakk** [comment]
microsoft#1125 now uses the functionality to use `".vfsforgit"` as the folder name on Linux.

**Comment by kivikakk** [comment]
Closing per other tickets merged into microsoft#1125.
